src/routes/pages.ts

```diff
@@ -0,0 +1,3 @@
+export const RESTAURANTS_PAGE = '/';
+export const SEARCH_PAGE = '/search';
+export const PROFILE_PAGE = '/profile';
```

Comment:
Please use enum instead

```suggestion
export enum PagePath {
  Restaurants = '/',
  Search = '/search',
  Profile = '/profile',
}
```